pyproject.toml

```diff
@@ -31,7 +30,7 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "<3.12,>=3.7.1"
+python = "<3.12,>=3.8.1"
```

Comment:
I took a quick peek here just to be sure not to many folks were on 3.7 and it looks fine to me to go with this!

https://pypistats.org/packages/meltanolabs-target-postgres